storey/flow.py

```diff
@@ -452,9 +553,21 @@ async def _do(self, event):
             return await self._do_downstream(_termination_obj)
         else:
             element = self._get_event_or_body(event)
-            fn_result = await self._call(element)
+            fn_result = await self._call(element, self._fn)
             await self._do_internal(event, fn_result)
 
+    async def select_outlets(self, event_body):
+        if self._outlets_selector:
+            return await self._call(event_body, self._outlets_selector, pass_kwargs=False)
```

Comment:
Seems to me that we should just call `self._outlets_selector()` directly? `self._call()` is for the actual work.